frontend/src/metabase/modes/components/drill/DashboardClickDrill.tsx

```diff
@@ -33,7 +41,7 @@ export default ({ question, clicked }) => {
     return [];
   }
   const { extraData } = clicked || {};
-  const data = getDataFromClicked(clicked);
+  const data = getDataFromClicked(clicked as any);
```

Comment:
Should we maybe extend `ClickActionProps` type's `extraData` type with what can actually be there?
Now it's `Record<string, any>`